Console/Command/ThemeChangeCommand.php

```diff
@@ -111,8 +111,8 @@ protected function execute(InputInterface $input, OutputInterface $output): int
             }
         }
 
-        if (!$scopeId > 0) {
-            $output->writeln('<error>Not a valid scopeId: ' . $scopeId . '</error>');
+        if (is_null($scopeId)) {
```

Comment:
Unfortunately, my previous commit has introduced an error which prevents setting a theme for the default scope.
Executing: `bin/magento theme:change <theme_name>` results in `Not a valid scopeId` error message.

Previously, I have added such condition in order to detect frontend scopes which have not been found.
I have now changed it so that when theme/store/website is not found `null` is returned instead.

Reply:
No worries, I'd rather have many PRs than be too careful - this module is just used by few, so we're fine messing up :)